gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/public/src/components/privacySettings/js/PrivacySettings.js

```diff
@@ -41,45 +41,96 @@ var gpii = gpii || {};
                 url: "src/core/available-authorized-preferences/%clientID.json"
             }
         },
+        members: {
+            // Keep track of tooltips created for edit and remove buttons.
+            // The tracked object elements are in pair of the button id and its tooltip component.
+            tooltips: {
```

Comment:
It appears that these members are only used from test cases. In that case, their definition and managing code should be moved into a grade defined with the test cases.